src/app/demo-form/demo-form.component.html

```diff
@@ -17,8 +17,7 @@ <h3>User profile form</h3>
 	<ckeditor
 		[(ngModel)]="model.description"
 		id="description"
-		name="description"
-		type="divarea">
+		name="description">
```

Comment:
This change will kill e2e tests. It should still use `divarea` plugin, otherwise, protractor won't be able to query DOM.